### ISSUE TITLE
removing leader for ai side

### ISSUE THREAD

**Issue by sevu** [ProditorMagnus/OrociaRandomMod#4]
Maybe `no_leader=yes` could be used instead of an leader for the AI side.

**Comment by ProditorMagnus** [comment]
Comment says leader is needed. Needs testing before removing.

**Comment by sevu** [comment]
`defeat_condition=no_units_left` for the human sides sound like it would fullfill the comments purpose.
Eventually `defeat_condition=never ` for the AI side.

**Comment by gfgtdf** [comment]
yes this can be done, i also remvoed the leader from the 2p dark forecast mainline surival in 1.13, `defeat_condition=never` for the AI side is needed.

**Comment by ProditorMagnus** [comment]
That leader is functionally not needed, though I once saw game with such settings that by turn 36 it had enough movement to pass though impassable and 10k damage.

But something like that would be better intentionally implemented, so will remove that leader soon.
